quickjs.h

```diff
@@ -591,44 +587,11 @@ JS_EXTERN JSValue __js_printf_like(2, 3) JS_ThrowInternalError(JSContext *ctx, c
 JS_EXTERN JSValue JS_ThrowOutOfMemory(JSContext *ctx);
 
 JS_EXTERN void __JS_FreeValue(JSContext *ctx, JSValue v);
-static inline void JS_FreeValue(JSContext *ctx, JSValue v)
-{
-    if (JS_VALUE_HAS_REF_COUNT(v)) {
-        JSRefCountHeader *p = (JSRefCountHeader *)JS_VALUE_GET_PTR(v);
-        if (--p->ref_count <= 0) {
-            __JS_FreeValue(ctx, v);
-        }
-    }
-}
+JS_EXTERN void JS_FreeValue(JSContext *ctx, JSValue v);
 JS_EXTERN void __JS_FreeValueRT(JSRuntime *rt, JSValue v);
```

Comment:
Let's drop this one too, since it's no longer used in the header file, is it?

Reply:
Nope, unless we want to keep for compatibility? But i suspect not as it requires JSRefCountHeader. Then we can also rename it to `js_free_value_rt` same story for `__JS_FreeValue`

Reply:
That one was only exposed so it could be used from the header file but kinda marked as "private" with the underscore. So yeah, they can go.